tests/test_tracking_grid/test_tracking_grid.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python
+
+import os
+import sys
+sys.path.insert(0, os.pardir)
+sys.path.insert(0, os.path.join(os.pardir, 'openmoc'))
+from testing_harness import TestHarness
+from input_set import GridInput
+
+import openmoc
+import math
+
+class TrackingGridTestHarness(TestHarness):
+    """Tests cell radial discretization."""
+
+    def __init__(self):
+        super(TrackingGridTestHarness, self).__init__()
+        self.input_set = GridInput()
+        self._result = ''
+
+
+    def _setup(self):
+        """Initialize the materials and geometry in the InputSet."""
+        super(TrackingGridTestHarness, self)._create_geometry()
+
+
+    def _segment_track(self, track, geometry):
+
+        # segmentize a track in a geometry, recording the segments in a string
+        geometry.segmentize(track)
+        num_segments = track.getNumSegments()
+        info = 'Number of segments = ' + str(num_segments) + '\n'
+        for i in range(num_segments):
+            info += 'Segment ' + str(i) + ': '
+            segment = track.getSegment(i)
+            info += 'length=' + str(round(segment._length, 8)) + ', '
+            info += 'FSR ID=' + str(segment._region_id) + ', '
+            info += 'CMFD FWD=' + str(segment._cmfd_surface_fwd) + ', '
+            info += 'CMFD BWD=' + str(segment._cmfd_surface_bwd) + ', '
+            info += 'Material Name=' + str(segment._material.getName()) + ', '
+            info += 'Material ID=' + str(segment._material.getId()) + '\n'
+        track.clearSegments()
+        return info
+
+    def _run_openmoc(self):
+
```

Comment:
Add a one line docstring to this method signature - specifically describe how this is different from the parent's `_run_openmoc()`